extensions/data-transfer/portability-data-transfer-rememberthemilk/build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
 
     compile("com.google.http-client:google-http-client-xml:${googleApiClient}")
     compile("com.google.api-client:google-api-client:${googleApiClient}")
+    compile group: 'com.fasterxml', name: 'jackson-xml-databind', version: '0.6.2'
```

Comment:
Can we used the compile() syntax to be consistent?

Reply:
Done.